### PR TITLE
Feature/dx 477 review and fix seamless view

### PR DESCRIPTION
**Pull request by trieutran91**
A lot of reorganizing and adding sections that are conform to the rest of the portal. 
Minor fixes of indent, hard cut-off, and links.

### CODE REVIEW

File: payments/swish/redirect.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ initiate the sales transaction.
 ## API Requests
 
 The API requests are displayed in the [purchase flow](#purchase-flow-3). Swish
-is a one-phase payment instrument that is based on sales transactions not involving
+is a one-phase payment method that is based on sales transactions not involving
```
Comment (asbjornu) [inline payments/swish/redirect.md]: Didn't we agree to use "instrument" in the documentation?

```suggestion
is a one-phase payment instrument that is based on sales transactions not involving
```

File: payments/swish/seamless-view.md [hunk 2]
```diff
@@ -18,13 +18,19 @@ sidebar:
 
 {% include alert-development-section.md %}
 
-> Swish is a one-phase payment instrument supported by the major Swedish banks. In
-  the direct e-commerce scenario, Swedbank Pay receives the Swish registered
-  mobile number directly from the merchant UI. Swedbank Pay performs a payment
-  that the payer confirms using her Swish mobile app.
+{% include jumbotron.html body="Swish is a one-phase payment instrument
+supported by the major Swedish banks. In the direct e-commerce scenario,
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]: Isn't *Seamless View* and *Direct* two different scenarios / integration variants?

File: payments/swish/seamless-view.md [hunk 3]
```diff
@@ -18,13 +18,19 @@ sidebar:
...
+Swedbank Pay receives the Swish registered mobile number directly from the
+merchant UI. Swedbank Pay performs a payment that the payer confirms using her
+Swish mobile app." %}
 
 ## Introduction
 
+The Seamless View integration provide you with the Swish payment solution
+directly on your website. This gives the consumer a frictionless experience as
+we are handling the payment in the implemented `iframe` on your page (see
+example below).
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]: Perhaps this paragraph can be used in the jumbotron above instead?

File: payments/swish/redirect.md (hunk 1 above)
Comment (trieutran91) [reply payments/swish/redirect.md]: We sure did! My bad!

File: payments/swish/seamless-view.md (hunk 3 above)
Comment (trieutran91) [reply payments/swish/seamless-view.md]: How about a soft intro  in the jumbotron and then use this text as a start to the section?

File: payments/swish/redirect.md [hunk 4]
```diff
@@ -143,8 +143,8 @@ sequenceDiagram
 
 ## Swish m-commerce Redirect
 
-> Swish is an one-phase payment instrument supported by the major Swedish banks. In
-  the redirect m-commerce scenario, Swedbank Pay performs a payment that the
+> Swish is an one-phase payment instrument supported by the major Swedish banks.
```
Comment (asbjornu) [inline payments/swish/redirect.md]: Please replace `>` with jumbotron if it makes sense.

File: payments/swish/seamless-view.md [hunk 5]
```diff
@@ -18,120 +18,389 @@ sidebar:
...
+      },
+      {
+        "method": "GET",
+        "href": "https://ecom.dev.payex.com/swish/core/scripts/client/px.swish.client.js?token=123456123412341234123456789012",
+        "rel": "view-sales",
+        "contentType": "application/javascript"
+      },
+      {
+            "method": "GET",
+            "href": "https://ecom.stage.payex.com/swish/core/scripts/client/px.swish.client.js?token=123456123412341234123456789012",
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]: All URLs should consistently refer to our `externalintegration` environment.

```suggestion
            "href": "https://ecom.externalintegration.payex.com/swish/core/scripts/client/px.swish.client.js?token=123456123412341234123456789012",
```

File: payments/swish/seamless-view.md [hunk 6]
```diff
@@ -18,120 +18,389 @@ sidebar:
...
-  activate SwedbankPay
-  SwedbankPay-->>-Merchant: Payment response
-  activate Merchant
-  Merchant-->>-Browser: Payment Status
+{% include intent.md %}
+
+### Operations
+
+The API requests are displayed in the purchase flow above.
+You can [create a card `payment`][create-payment] with following `operation`
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]:
```suggestion
You can [create a Swish `payment`][create-payment] with following `operation`
```

File: payments/swish/seamless-view.md [hunk 7]
```diff
@@ -18,120 +18,389 @@ sidebar:
...
+        },
+        "payeeInfo": {
+            "payeeId": "12345678-1234-1234-1234-123456789012",
+            "payeeReference": "CD1234",
+            "payeeName": "Merchant1",
+            "productCategory": "A123",
+            "orderReference": "or123",
+        },
+        "prefillInfo": {
+          "msisdn": "{{msisdn}}"
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]: Where is the `msisdn` variable defined?

Comment (NullableInt) [inline payments/swish/seamless-view.md]: This could be replaced with test data.

File: payments/swish/seamless-view.md [hunk 8]
```diff
@@ -18,120 +18,389 @@ sidebar:
...
-  Merchant-->>-Browser: Payment Status
+{% include intent.md %}
+
+### Operations
+
+The API requests are displayed in the purchase flow above.
+You can [create a card `payment`][create-payment] with following `operation`
+options:
+
+* [Purchase][purchase] (We use this value in our examples)
```
Comment (NullableInt) [inline payments/swish/seamless-view.md]: A bullet list containing only one element seems redundant. Perhaps rewrite the top section to show there are no options?

File: payments/swish/seamless-view.md [hunk 9]
```diff
@@ -18,120 +18,389 @@ sidebar:
...
+        "rel": "update-payment-abort",
+        "method": "PATCH",
+        "contentType": "application/json"
+      },
+      {
+            "method": "POST",
+            "href": "https://api.stage.payex.com/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales",
+            "rel": "create-sale"
+        },
+      {
```
Comment (NullableInt) [inline payments/swish/seamless-view.md]: This bracket is oddly indented.

Comment (trieutran91) [reply payments/swish/seamless-view.md]: Well spotted! Fixed!

File: payments/swish/seamless-view.md (hunk 8 above)
Comment (trieutran91) [reply payments/swish/seamless-view.md]: Redundant indeed. Better now!

File: payments/swish/seamless-view.md (hunk 7 above)
Comment (trieutran91) [reply payments/swish/seamless-view.md]: Forgot to change the name of the property in the table

File: payments/swish/redirect.md (hunk 4 above)
Comment (trieutran91) [reply payments/swish/redirect.md]: I'll leave this for the redirect branch to handle.

File: payments/swish/seamless-view.md (hunk 3 above)
Comment (asbjornu) [reply payments/swish/seamless-view.md]: Sure

File: payments/swish/seamless-view.md [hunk 10]
```diff
@@ -18,120 +18,386 @@ sidebar:
 
 {% include alert-development-section.md %}
 
-> Swish is a one-phase payment instrument supported by the major Swedish banks. In
-  the direct e-commerce scenario, Swedbank Pay receives the Swish registered
-  mobile number directly from the merchant UI. Swedbank Pay performs a payment
-  that the payer confirms using her Swish mobile app.
+{% include jumbotron.html body="The Seamless View scenario
+                          represents the opportunity to implement Swish payments
+                          directly in your webshop." %}
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]: Use similar wording as suggested in #109.

File: payments/swish/seamless-view.md [hunk 11]
```diff
@@ -18,120 +18,386 @@ sidebar:
...
+        SwedbankPay->>SwedbankPay: Payment is updated
+        SwedbankPay->>-Merchant: POST Payment Callback
+        end
+```
+
+### Explainations
+
+* ① When the payer starts the purchase process, you make a `POST` request
+  towards Swedbank Pay with the collected Purchase information.
+* ② `rel: view-sales` is a value in one of the operations, sent as a response
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]:
```suggestion
* ② `view-sales` is a `rel` value in one of the operations, sent as a response
```

File: payments/swish/seamless-view.md [hunk 12]
```diff
@@ -18,120 +18,386 @@ sidebar:
...
+  cancelled payment.
+* ⑥ To get the transaction result, you need to follow up with a `GET` request
+  using the `paymentID` received in the first step.
+* ⑦ If CallbackURL is set you will receive a payment callback when the Swish
   dialogue is completed, and you will have to make a `GET` request to check the
   payment status.
-* The flow is explained in the sequence diagram below.
 
-## API Requests
+### 3-D Secure
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]: As for Vipps, I don't think 3-D Secure is relevant for Swish payments. Perhaps when you register for Swish the first time, but not when performing payments.

File: payments/swish/seamless-view.md [hunk 13]
```diff
@@ -18,120 +18,386 @@ sidebar:
...
+            "deliveryEmailAddress": "string",
+            "deliveryTimeFrameindicator": "01",
+            "preOrderDate": "YYYYMMDD",
+            "preOrderPurchaseIndicator": "01",
+            "shipIndicator": "01",
+            "giftCardPurchase": false,
+            "reOrderPurchaseIndicator": "01",
+            "pickUpAddress": {
+                "name": "companyname",
+                "streetAddress": "string",
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]: Please use relevant test data

File: payments/swish/seamless-view.md [hunk 14]
```diff
@@ -18,120 +18,386 @@ sidebar:
...
+      "captures": { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/captures" },
+      "reversals": { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/reversals" },
+      "cancellations": { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/cancellations" },
+      "urls" : { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/urls" },
+      "payeeInfo" : { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/payeeInfo" },
+      "settings": { "id": "/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/settings" }
+    },
+    "operations": [
+      {
+        "href": "https://api.payex.com/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
```
Comment (asbjornu) [inline payments/swish/seamless-view.md]: Always use `externalintegration.payex.com` in example URLs.

```suggestion
        "href": "https://api.externalintegration.payex.com/psp/swish/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
```

File: payments/swish/seamless-view.md [hunk 15]
```diff
@@ -18,120 +18,371 @@ sidebar:
...
+|  ✔︎︎︎︎︎  | └➔&nbsp;`currency`                    | `string`      | NOK, SEK, DKK, USD or EUR.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (NullableInt) [inline payments/swish/seamless-view.md]: Only `SEK` for Swish

Comment (asbjornu) [inline payments/swish/seamless-view.md]:
```suggestion
|  ✔︎︎︎︎︎  | └➔&nbsp;`currency`                    | `string`      | `SEK`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

**Pull request merged**